[Panic] When upgrading to latest codegen in pulumi v2.5.0

```
panic: runtime error: slice bounds out of range [2:1]

goroutine 1 [running]:
github.com/pulumi/pulumi/pkg/v2/codegen/schema.shortcodeParser.parseShortcode(0x0, 0xc0013798cf, 0x1, 0x231, 0x2, 0x250, 0x0, 0xc002058000, 0x0)
	/Users/stack72/code/go/pkg/mod/github.com/pulumi/pulumi/pkg/v2@v2.5.1-0.20200701223250-45d2fa95d60b/codegen/schema/docs_parser.go:66 +0x3bc
github.com/pulumi/pulumi/pkg/v2/codegen/schema.shortcodeParser.Continue(0x0, 0x6f49f80, 0xc001f7bad0, 0x6f3ee20, 0xc001edfe30, 0x6f40f20, 0xc001edfea0, 0x231)
	/Users/stack72/code/go/pkg/mod/github.com/pulumi/pulumi/pkg/v2@v2.5.1-0.20200701223250-45d2fa95d60b/codegen/schema/docs_parser.go:142 +0xb2
github.com/pgavlin/goldmark/parser.(*parser).parseBlocks(0xc002229500, 0x6f487c0, 0xc001ff6e60, 0x6f3ee20, 0xc001edfe30, 0x6f40f20, 0xc001edfea0)
	/Users/stack72/code/go/pkg/mod/github.com/pgavlin/goldmark@v1.1.33-0.20200616210433-b5eb04559386/parser/parser.go:1072 +0x4fc
github.com/pgavlin/goldmark/parser.(*parser).Parse(0xc002229500, 0x6f3ee20, 0xc001edfe30, 0x0, 0x0, 0x0, 0xc002938d80, 0x402)
	/Users/stack72/code/go/pkg/mod/github.com/pgavlin/goldmark@v1.1.33-0.20200616210433-b5eb04559386/parser/parser.go:848 +0x142
github.com/pulumi/pulumi/pkg/v2/codegen/schema.ParseDocs(0xc001379680, 0x402, 0x480, 0xc001379680, 0x402)
	/Users/stack72/code/go/pkg/mod/github.com/pulumi/pulumi/pkg/v2@v2.5.1-0.20200701223250-45d2fa95d60b/codegen/schema/docs_parser.go:173 +0x16d
github.com/pulumi/pulumi/pkg/v2/codegen.FilterExamples(0xc002938d80, 0x402, 0x6765693, 0xa, 0xc000a012b0, 0xc0032bc080)
	/Users/stack72/code/go/pkg/mod/github.com/pulumi/pulumi/pkg/v2@v2.5.1-0.20200701223250-45d2fa95d60b/codegen/docs.go:94 +0x93
github.com/pulumi/pulumi/pkg/v2/codegen/nodejs.(*modContext).genResource(0xc001310dd0, 0x6e0c3c0, 0xc00201c7b0, 0xc001846b40, 0x2, 0x2)
	/Users/stack72/code/go/pkg/mod/github.com/pulumi/pulumi/pkg/v2@v2.5.1-0.20200701223250-45d2fa95d60b/codegen/nodejs/gen.go:369 +0x8b
github.com/pulumi/pulumi/pkg/v2/codegen/nodejs.(*modContext).gen(0xc001310dd0, 0xc0007fba40, 0x0, 0x0)
	/Users/stack72/code/go/pkg/mod/github.com/pulumi/pulumi/pkg/v2@v2.5.1-0.20200701223250-45d2fa95d60b/codegen/nodejs/gen.go:1095 +0x357
github.com/pulumi/pulumi/pkg/v2/codegen/nodejs.GeneratePackage(0x67bf48e, 0x28, 0xc002966120, 0xc00166c1e0, 0xb, 0xc00166c1e0, 0x0)
	/Users/stack72/code/go/pkg/mod/github.com/pulumi/pulumi/pkg/v2@v2.5.1-0.20200701223250-45d2fa95d60b/codegen/nodejs/gen.go:1549 +0x4b6
github.com/pulumi/pulumi-terraform-bridge/v2/pkg/tfgen.language.emitSDK(0x7ffeefbff8c7, 0x6, 0xc002966120, 0xc0007ab500, 0x675f566, 0x8, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/stack72/code/go/pkg/mod/github.com/pulumi/pulumi-terraform-bridge/v2@v2.5.3/pkg/tfgen/generate.go:99 +0x28f
github.com/pulumi/pulumi-terraform-bridge/v2/pkg/tfgen.(*generator).Generate(0xc000720ea0, 0x8, 0x6da3ed0)
	/Users/stack72/code/go/pkg/mod/github.com/pulumi/pulumi-terraform-bridge/v2@v2.5.3/pkg/tfgen/generate.go:658 +0x3bd
github.com/pulumi/pulumi-terraform-bridge/v2/pkg/tfgen.newTFGenCmd.func1(0xc0002c9080, 0xc000472eb0, 0x1, 0x5, 0x0, 0x0)
	/Users/stack72/code/go/pkg/mod/github.com/pulumi/pulumi-terraform-bridge/v2@v2.5.3/pkg/tfgen/main.go:78 +0x197
github.com/pulumi/pulumi/sdk/v2/go/common/util/cmdutil.RunFunc.func1(0xc0002c9080, 0xc000472eb0, 0x1, 0x5, 0xc000b1fa08, 0x1b59ce4)
	/Users/stack72/code/go/pkg/mod/github.com/pulumi/pulumi/sdk/v2@v2.5.1-0.20200701223250-45d2fa95d60b/go/common/util/cmdutil/exit.go:96 +0x51
github.com/pulumi/pulumi/sdk/v2/go/common/util/cmdutil.RunResultFunc.func1(0xc0002c9080, 0xc000472eb0, 0x1, 0x5)
	/Users/stack72/code/go/pkg/mod/github.com/pulumi/pulumi/sdk/v2@v2.5.1-0.20200701223250-45d2fa95d60b/go/common/util/cmdutil/exit.go:112 +0x6b
github.com/spf13/cobra.(*Command).execute(0xc0002c9080, 0xc000070250, 0x5, 0x5, 0xc0002c9080, 0xc000070250)
	/Users/stack72/code/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:846 +0x29d
github.com/spf13/cobra.(*Command).ExecuteC(0xc0002c9080, 0x8, 0x6da3ed0, 0xc)
	/Users/stack72/code/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x349
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/stack72/code/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
github.com/pulumi/pulumi-terraform-bridge/v2/pkg/tfgen.Main(0x675f566, 0x8, 0x6da3ed0, 0xc, 0xc0007ab500, 0x675f566, 0x8, 0x0, 0x0, 0x0, ...)
	/Users/stack72/code/go/pkg/mod/github.com/pulumi/pulumi-terraform-bridge/v2@v2.5.3/pkg/tfgen/main.go:31 +0xa8
main.main()
	/Users/stack72/code/go/src/github.com/pulumi/pulumi-alicloud/provider/cmd/pulumi-tfgen-alicloud/main.go:24 +0xc1
make: *** [build_nodejs] Error 2
```

The branch is https://github.com/pulumi/pulumi-alicloud/tree/go-examples-2

--- a/pkg/codegen/schema/docs_parser.go
+++ b/pkg/codegen/schema/docs_parser.go
@@ -133,9 +133,11 @@ func (p shortcodeParser) Open(parent ast.Node, reader text.Reader, pc parser.Con
 }
 
 func (p shortcodeParser) Continue(node ast.Node, reader text.Reader, pc parser.Context) parser.State {
-	line, _ := reader.PeekLine()
+	line, seg := reader.PeekLine()
 	pos := pc.BlockOffset()
 	if pos < 0 {
+		return parser.Continue | parser.HasChildren
+	} else if pos > seg.Len() {
 		return parser.Continue | parser.HasChildren
 	}
 

